announcer.js

```diff
@@ -7,47 +7,66 @@ const base = new Airtable({
     apiKey: process.env.AIRTABLE_API_KEY
 }).base(process.env.AIRTABLE_BASE);
 
-base('Slack Announcer').select({
-    view: "Announcer Filter"
-}).eachPage(function page(records, fetchNextPage) {
+var airtableCronJobs = [];
 
-    // This function (`page`) will get called for each page of records.
+// Flush and reload the CRON table at 3 minutes and 3 seconds past every hour
+// This is specifically offset from 5, 10, 15 minute intervals to ensure that 
```

Comment:
How do you feel about looking for anything from like 2 minutes 0 seconds to 4 minutes 59 seconds and dynamically changing it to 5 minutes after the hour? I could output a console log whenever the script does this.